src/tools/kata-ctl/src/check.rs

```diff
@@ -25,6 +34,8 @@ const USER_AGT: &str = "kata";
 
 #[allow(dead_code)]
 const ERR_NO_CPUINFO: &str = "cpu_info string is empty";
+const ERR_LSMOD: &str = "check_kernel_module_loaded return:";
+const ERR_MODPROBE: &str = "check_kernel_module_loaded return: module not loaded:";
```

Comment:
Can you remove the function names from these errors [*]. I'm not totally sure we need these consts as they don't specify the actual error ("module not found", "module not loaded"?)

---

[*] - it's frustrating that rust _still_ doesn't provide an equivalent to C's `__func__`, although it does provide `file!()` and `line!()`.